File: examples/url/proxy/go/resolveAShortUrlToALongDestinationUrl.go
package example

import (
	"fmt"
	"github.com/micro/services/clients/go/url"
	"os"
)

// Proxy returns the destination URL of a short URL.
func ResolveAshortUrlToAlongDestinationUrl() {
	urlService := url.NewUrlService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := urlService.Proxy(&url.ProxyRequest{})
	fmt.Println(rsp, err)
}

File: examples/url/proxy/node/resolveAShortUrlToALongDestinationUrl.js
import * as url from "m3o/url";

// Proxy returns the destination URL of a short URL.
async function ResolveAshortUrlToAlongDestinationUrl() {
  let urlService = new url.UrlService(process.env.MICRO_API_TOKEN);
  let rsp = await urlService.proxy({
    shortURL: "https://m3o.one/u/ck6SGVkYp",
  });
  console.log(rsp);
}

await ResolveAshortUrlToAlongDestinationUrl();
